### PR TITLE
Add inifinite sum

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: src/makeInfinityAdder.js [hunk 1]
```diff
@@ -6,7 +6,22 @@
...
+      const result = count;
+
+      count = 0;
+
+      return result;
+    } else {
+      count += numbers;
+
+      return adder;
+    }
```
Comment (illiarerun) [inline src/makeInfinityAdder.js:21]: you don't need else here, anyways you return value from previous if

```suggestion
    }
   
    count += numbers;

    return adder;
```

File: src/makeInfinityAdder.js [hunk 2]
```diff
@@ -6,7 +6,22 @@
  */
 
 function makeInfinityAdder() {
-  // write code here
+  let count = 0;
```
Comment (illiarerun) [inline src/makeInfinityAdder.js:9]: that's not a `count`, it's a sum

File: src/makeInfinityAdder.js [hunk 3]
```diff
@@ -6,7 +6,22 @@
...
+  const adder = (numbers) => {
```
Comment (illiarerun) [inline src/makeInfinityAdder.js:10]: it's only 1 number, why you're using plural name here?
```suggestion
  let count = 0;
  
  const adder = (number) => {
```

**Review by illiarerun — CHANGES_REQUESTED** [review]
GJ, let's work on fixes now:

**Review by jstmpelowycz — APPROVED** [review]

**Pull request closed without merging**